framework/resource_router.go

```diff
@@ -37,10 +37,10 @@ func (r *ResourceRouter) ResourceSet(obj interface{}) (*ResourceSet, error) {
 	}
 
 	if len(found) == 0 {
-		return nil, microerror.Maskf(executionFailedError, "could not find any resource router")
+		return nil, microerror.Maskf(executionFailedError, "could not find handling resource set")
```

Comment:
Nitpick, but for me this somehow sounds weird. What would think of following:
`no match for resource set`

Reply:
This means that resource set that is capable of handling custom object that just arrived wasn't found.

I guess is sounds more reasonable when you are into the details, doesn't it?

I'd be glad to make it something reasonable for a newcomer. `handling resource set not found` makes more sense?

Reply:
Yeah, I actually did know what this is used for. Your new suggestion sounds very good. I like it :+1: